.travis.yml

```diff
@@ -41,7 +41,6 @@ addons:
       - libgsm1-dev
       - libavcodec-dev
       - libavutil-dev
-      - libx264-dev
```

Comment:
that might result in problems with `ffmpeg` backend which uses `x264` internally